src/TurbulenceClosures/turbulence_closure_implementations/CATKEVerticalDiffusivities/CATKEVerticalDiffusivities.jl

```diff
@@ -107,37 +110,13 @@ const FlavorOfCATKE{TD} = Union{CATKEVD{TD}, CATKEVDArray{TD}} where TD
 include("mixing_length.jl")
 include("turbulent_kinetic_energy_equation.jl")
 
-# Optimal parameters for "favorite CATKE" from Wagner et al. 2023 (in prep)
-optimal_turbulent_kinetic_energy_equation(FT) = TurbulentKineticEnergyEquation(
-    C⁻D  = FT(4.4),
-    C⁺D  = FT(3.3),
-    CᶜD  = FT(0.23),
-    CᵉD  = FT(0.0),
-    Cᵂu★ = FT(1.8),
-    CᵂwΔ = FT(12.0))
-
-optimal_mixing_length(FT) = MixingLength(
-    Cᵇ   = FT(0.37), 
-    Cᶜc  = FT(4.8),
-    Cᶜe  = FT(1.1),
-    Cᵉc  = FT(0.049),
-    Cᵉe  = FT(0.0),
-    Cˢᶜ  = FT(0.29),
-    C⁻u  = FT(0.36),
-    C⁺u  = FT(0.24),
-    C⁻c  = FT(0.41),
-    C⁺c  = FT(0.12),
-    C⁻e  = FT(6.7),
-    C⁺e  = FT(5.4),
-    CRiʷ = FT(0.011),
-    CRiᶜ = FT(0.76))
-
-function CATKEVerticalDiffusivity(time_discretization::TD = VerticallyImplicitTimeDiscretization(), FT=Float64;
-                                  mixing_length = optimal_mixing_length(FT),
-                                  turbulent_kinetic_energy_equation = optimal_turbulent_kinetic_energy_equation(FT),
+function CATKEVerticalDiffusivity(time_discretization::TD = VerticallyImplicitTimeDiscretization(),
+                                  FT = Float64;
+                                  mixing_length = MixingLength(),
+                                  turbulent_kinetic_energy_equation = TurbulentKineticEnergyEquation(),
                                   maximum_diffusivity = Inf,
-                                  minimum_turbulent_kinetic_energy = 0,
-                                  minimum_convective_buoyancy_flux = 1e-11,
+                                  minimum_turbulent_kinetic_energy = 1e-6,
+                                  minimum_convective_buoyancy_flux = 1e-8,
                                   negative_turbulent_kinetic_energy_damping_time_scale = 1minute,
                                   warning = true) where TD
```

Comment:
```suggestion
                                  warning = false) where TD
```

Reply:
Or keep the warning but change it to say that it's "a new closure calibrated with LES; see paper doi:..."

Personally I think nuke the warning all together and add some of this info (about calibration and more info look at preprint at doi:....) in the docstring.

Reply:
I pushed something; have a look!

Reply:
Good